app/controllers/concerns/remember_device_concern.rb

```diff
@@ -6,7 +6,7 @@ def save_user_opted_remember_device_pref
   end
 
   def save_remember_device_preference
-    return unless params[:remember_device] == 'true'
+    return unless params[:remember_device] == '1' || params[:remember_device] == 'true'
```

Comment:
continuing my personal beef with `unless`:

```suggestion
    return if params[:remember_device] != '1' && params[:remember_device] != 'true'
```